### PR TITLE
rework wording of file management

### PR DESCRIPTION
**Pull request by Pete-Y-CS**

### CODE REVIEW

File: src/lib/files/FileManager.svelte [hunk 1]
```diff
@@ -18,6 +18,7 @@
   // eslint-disable-next-line no-undef
   export let state: Writable<StateType>;
   export let service: string;
+  export let fileObjectType: string;
```
Comment (dabreegster) [inline src/lib/files/FileManager.svelte:21]: Comment `// A singular noun like "scheme". The plural noun is assumed to just need an "s" appended`

File: src/lib/files/FileManager.svelte [hunk 2]
```diff
@@ -125,16 +126,16 @@
...
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <slot name="description" />
 
       <p>
-        All files are auto-saved in your browser's local storage. You can close
-        your browser and resume later. You can export the file to your computer
-        to share with someone else, and import from a file below.
+        On this page you can manage your assessment files. All files are
```
Comment (dabreegster) [inline src/lib/files/FileManager.svelte:136]: "assessment files" sounds planning-specific

File: src/lib/files/FileManager.svelte [hunk 3]
```diff
@@ -125,16 +126,16 @@
...
+        auto-saved in your browser's local storage. You can close your browser
+        and conitnue later.
```
Comment (dabreegster) [inline src/lib/files/FileManager.svelte:138]: Typo, continue

File: src/lib/files/FileManager.svelte [hunk 4]
```diff
@@ -187,12 +188,10 @@
     </div>
 
     <div class="govuk-grid-column-one-third">
-      <h2 class="green-bar">Create or import a file</h2>
-      <SecondaryButton on:click={newFile}>
-        New blank scheme file
-      </SecondaryButton>
-      <hr />
-      <FileInput label="Import from a .json file" onLoad={importJsonFile} />
+      <h2 class="green-bar">Create a new {fileObjectType}</h2>
```
Comment (dabreegster) [inline src/lib/files/FileManager.svelte:191]: Could we check the final result with Jadene? Heading "Create a new cross section", button "Create new file" beneath it. It feels a bit like we're using both bits of terminology here.

**Review by dabreegster — CHANGES_REQUESTED** [review]

File: src/lib/files/FileManager.svelte (hunk 2 above)
Comment (Pete-Y-CS) [reply src/lib/files/FileManager.svelte:136]: oops good spot

File: src/lib/files/FileManager.svelte (hunk 4 above)
Comment (Pete-Y-CS) [reply src/lib/files/FileManager.svelte:191]: This is from the design but agree it's a bit strange. Will push the other changes momentarily then we can demo to Jadene

**Pull request merged**